config/services.xml

```diff
@@ -8,6 +8,8 @@
         <service id="doctrine.fixtures_load_command" class="Doctrine\Bundle\FixturesBundle\Command\LoadDataFixturesDoctrineCommand">
             <argument type="service" id="doctrine.fixtures.loader" />
             <argument type="service" id="doctrine" />
+            <argument type="collection"></argument>
+            <argument type="service" id="logger" on-invalid="null"/>
```

Comment:
I don't know if this is the right logger instance to use here, @derrabus maybe you will know.

Reply:
This is fine, I guess. In case, the app uses Monolog, we should subscribe to the Doctrine channel.

```xml
<tag name="monolog.logger" channel="doctrine" />
```

Reply:
Ah thanks, that was the missing piece 👍